indico/modules/events/models/events.py

```diff
@@ -244,6 +244,13 @@ def __table_args__(cls):
         index=True,
         nullable=True
     )
+    #: The url to a map for the event
+    _map_url = db.Column(
```

Comment:
what about `own_map_url` for the column attr (`map_url` in the DB), and `map_url` for the property?